docs/proposals/20210722-yurtcluster-operator.md

```diff
@@ -72,43 +74,174 @@ This is an operator, with CRD and Controller.
 CRD's definition:
 
 ```go
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// ImageMeta allows to customize the image used for components that are not
+// originated from the openyurtio/openyurt release process
+type ImageMeta struct {
+	// ImageRepository sets the container registry to pull images from.
+	// if not set, the ImageRepository defined in YurtClusterSpec will be used instead.
+	// +optional
+	ImageRepository string `json:"imageRepository,omitempty"`
+	// ImageTag allows to specify a tag for the image.
+	// if not set, the tag releated to the YurtVersion defined in YurtClusterSpec will be used instead.
+	// +optional
+	ImageTag string `json:"imageTag,omitempty"`
```

Comment:
How about remove the prefix `Image`? 
so `ImageRepository ` --> `Repository`, `ImageTag` --> `Tag`